packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js

```diff
@@ -36,7 +37,7 @@ export const mapFilterForIceberg = (comparator, reference) =>
 export const getLink = (column, tracker) => `
   <a
     data-ng-href="{{ $ctrl.getServiceNameLink($row) }}"
-    data-ng-bind="$row.${column.property}"
+    data-ng-bind="$ctrl.convertDomainToPunycode($row.${column.property})"
```

Comment:
This column can be other things that a Domain. Maybe it is better to change the function name ?

Reply:
But I wanted to keep it consistent with original namings (could be usefull in the future to have same function name with same intention everywhere in the code).

What do you think?

Reply:
Maybe you can change it everywhere by "convertToPunycode ?